src/uct/ib/base/ib_md.c

```diff
@@ -185,6 +185,11 @@ static ucs_config_field_t uct_ib_md_config_table[] = {
      "Enable setting CPU affinity of memory registration threads.",
      ucs_offsetof(uct_ib_md_config_t, ext.mt_reg_bind), UCS_CONFIG_TYPE_BOOL},
 
+    {"PCI_RELAXED_ORDERING", "auto",
```

Comment:
looks like it is off by default. Maybe set `off` as a default value?

Reply:
we want to make "auto" detect by platform type